tests/web/rpc/test_aggregation.py

```diff
@@ -0,0 +1,52 @@
+from snuba.web.rpc.common.aggregation import (
+    CUSTOM_COLUMN_PREFIX,
+    CustomColumnInformation,
+    _generate_custom_column_alias,
+    get_custom_column_information,
+)
+
+
+def test_generate_custom_column_alias() -> None:
+    custom_column_information_with_metadata = CustomColumnInformation(
+        column_type="column_type",
+        referenced_column="count",
+        metadata={"meta1": "value1", "meta2": "value2"},
```

Comment:
this doesn't give me a good indication of what this metadata column would be used for